src/main/java/com/urswolfer/intellij/plugin/gerrit/push/GerritPushExtensionPanel.java

```diff
@@ -130,6 +130,10 @@ public void run() {
                     Properties properties = new Properties();
                     properties.load(fileInputStream);
                     branchName = Optional.fromNullable(properties.getProperty("defaultbranch"));
```

Comment:
Guava Strings#emptyToNull probably would simplify this code a little bit.

Reply:
Used Strings.isNullOrEmpty as more appropriate one

Reply:
Oh, it seems I misunderstood what to change firstly. Now I used Strings#emptyToNull and it really simplified the code. Thank you veru much for your suggestion!